plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/connection/DockerConnectionFactory.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A. All rights reserved. This program and the accompanying materials are made available under the terms
+ * of the Eclipse Public License v1.0 which accompanies this distribution, and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.connection;
+
+import com.google.inject.Inject;
+
+import org.eclipse.che.plugin.docker.client.DockerCertificates;
+import org.eclipse.che.plugin.docker.client.DockerConnector;
+import org.eclipse.che.plugin.docker.client.DockerConnectorConfiguration;
+
+import javax.inject.Named;
+import java.net.URI;
+
+/**
+ * Factory for connections to docker API.<br>
```

Comment:
I am not sure about `<br>` here, i think it is better to use `<p>` in this case.
See [example](https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#paragraphs).